apps/people/src/app/page.tsx

```diff
@@ -1,14 +1,456 @@
-import { Stack } from '@jsxcss/emotion'
+'use client'
+
+import { useFloating, useHover, useInteractions } from '@floating-ui/react'
+import { Box, Flex, Stack } from '@jsxcss/emotion'
+import { Suspense } from '@suspensive/react'
+import { SuspenseImage } from '@suspensive/react-image'
+import { Reorder, motion, useMotionValue, useTransform } from 'framer-motion'
+import { useEffect, useState } from 'react'
 
 export default function Home() {
+  const [cards, setCards] = useState(initialCards)
+  const [selectedCardName, setSelectedCardName] = useState<Card['name'] | null>(
+    null
+  )
+
   return (
-    <div>
-      <Stack spacing={10}>
-        <div>hi</div>
-        <div>hi</div>
-        <div>hi</div>
-        <div>hi</div>
+    <Stack
+      overflow="hidden"
+      backgroundColor="#13120f"
+      height="100vh"
+      as={motion.div}
+      spacing={10}
+    >
+      <Flex.Center flex={1}>
+        <Suspense clientOnly fallback={null}>
+          <Suspense
+            clientOnly
+            fallback={
+              <Flex.Center
+                width={320}
+                height={320}
+                overflow="hidden"
+                borderRadius="50%"
+              >
```

Comment:
Suspense `clientOnly` 상위에만 걸면 다르게 동작하나요 ?.?

```suggestion
        <Suspense clientOnly fallback={null}>
          <Suspense
            fallback={
              <Flex.Center
                width={320}
                height={320}
                overflow="hidden"
                borderRadius="50%"
              >
```

Reply:
clientOnly가 있으면 서버사이드렌더링에서 이 부분을 스킵해요. 그래서 나중에 이부분의 seo를 위한 처리가 필요합니담

Reply:
음 제가 말한건 그게 아니라 `clientOnly`가 중복으로 있어서용  상위에만 달면 하위도 `clientOnly`가 되는거 아닌가하는 질문이였슴당 ㅎㅎ 코드체인지에 그래서 제가 하위에는 `clientOnly`를 지웠던! ㅋㅋ `Suspensive` 라이브러리 질문이라 넘어가도됩니당 ㅋㅋ

Reply:
아아 맞네요 하위에서 clientOnly가 필요 없을 거 같아요 리뷰 감사합니다